eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.js

```diff
@@ -183,15 +183,15 @@ export const UnwrappedMainNavigation = ({
               </RouteButton>
               <RouteButton
                 variant={
-                  (whatPageAreWeOn === "metadata" && "navigation-on") ||
+                  (whatPageAreWeOn === "data" && "navigation-on") ||
                   "navigation"
                 }
                 small
-                data-test="btn-metadata"
-                to={buildMetadataPath(params)}
+                data-test="btn-data"
+                to={buildDataPath(params)}
               >
                 <IconText nav icon={metadataIcon}>
```

Comment:
Might be worth renaming `metadataIcon` to `dataIcon`

Reply:
Made the changes